ccd-definition/ComplexTypes.json

```diff
@@ -1556,5 +1556,206 @@
     "FieldType": "Document",
     "ElementLabel": "Upload a file",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "telephone",
```

Comment:
I'd like to have case type names in camel case format like we do for classes in Java. Please rename it here and in other places.

Reply:
Yes King